ansible/ubuntu.yml

```diff
@@ -0,0 +1,72 @@
+---
+- hosts: 127.0.0.1
+  connection: local
+  become: true
+  become_user: root
+  
+  vars:
+    apt_key_url: hkp://p80.pool.sks-keyservers.net:80
+    apt_key_sig: 58118E89F3A912897C070ADBF76221572C52609D
+    apt_repository: deb https://apt.dockerproject.org/repo {{ ansible_lsb.id|lower }}-{{ ansible_lsb.codename|lower }} main
+
+  tasks:
+      
+    - name: Add Docker repo key
+      shell: "apt-key adv --keyserver {{ apt_key_url }} --recv-keys {{ apt_key_sig }}"
+
+    - name: HTTPS APT transport for Docker repository
+      apt:
+        name: apt-transport-https
+        state: present
+
+    - name: Add docker repository to apt lists
+      apt_repository:
+        repo: "{{ apt_repository }}"
+        mode: '644'
+        update_cache: yes
+        state: present
+
+    - name: Install the base required dependencies and suggested packages
+      apt:
+        pkg:
+          - "linux-image-extra-{{ ansible_kernel }}"
+          - linux-image-extra-virtual
+          - dnsmasq
+          - apt-transport-https
+          - ca-certificates
+          - docker-engine
+          - git
+        state: latest
+        update_cache: yes
+
+    - name: Allow current user to execute docker commands
+      user: name={{ ansible_ssh_user }} groups=docker append=yes
+
+    - name: Check if docker-compose binary exists
+      stat: path=/usr/local/bin/docker-compose
+      register: docker_compose_bin
+
+    - name: Install docker-compose
+      shell: "curl -L https://github.com/docker/compose/releases/download/1.8.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose && chmod +x /usr/local/bin/docker-compose"
```

Comment:
I missed this increase in version... I guess @edodusi did upgrade since the playbook states 1.7.0.
No problem: which version should we stick to?

We can anyway have a common property file with such parameters to be configured for both OSes... I'll do.